Stylesheets/components/badge.css

```diff
@@ -0,0 +1,79 @@
+@import url(/Stylesheets/main.css);
+.container {
+}
+
+.mark {
+  margin: 2rem;
+}
+
+.bedge {
+  height: 3rem;
+  width: 3rem;
+}
+
+.badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  height: 2rem;
+  width: 2rem;
+  background-color: red;
+  border-radius: 50%;
+  position: absolute;
+}
+
+.bot {
+  bottom: 34.7rem;
+}
+
+.mail {
+  left: 4rem;
+}
+
+.cart {
+  left: 11rem;
+}
+.phone {
+  left: 18rem;
+}
+.insta {
+  left: 25rem;
+}
+
+.hero-img {
+  height: 100px;
+  width: auto;
+  border-radius: 50%;
+}
+
+.social {
+  height: 2rem;
+  width: 2rem;
+  border: 0.2rem solid white;
```

Comment:
keep consistency throughout the code and use variables which you have declared previously